src/utils.ts

```diff
@@ -53,7 +53,16 @@ export const isMatchingStartUrl = (
   cred: CachedCredential,
   profile: Profile
 ): boolean => {
-  const isMatch = cred.startUrl === profile?.sso_start_url;
+  let isMatch = false;
```

Comment:
Instead of using the optional chaining which is causing the coverage to complain; run the check at the top of the function to reduce overhead.

ie:
```ts
  if (!profile || !profile.sso_start_url) {
    return false;
  }

```

Then remove the optional chaining from the checks below:

ie:
```ts
profile?.sso_start_url.length
```

to
```ts
profile.sso_start_url.length
```